java/adapter/avro/src/main/java/org/apache/arrow/consumers/AvroIntConsumer.java

```diff
@@ -21,24 +21,37 @@
 
 import org.apache.arrow.vector.IntVector;
 import org.apache.arrow.vector.complex.impl.IntWriterImpl;
-import org.apache.arrow.vector.complex.writer.IntWriter;
 import org.apache.avro.io.Decoder;
 
 /**
  * Consumer which consume int type values from avro decoder.
  * Write the data to {@link IntVector}.
  */
-public class AvroIntConsumer implements Consumer {
+public class AvroIntConsumer extends Consumer {
 
-  private final IntWriter writer;
+  private final IntWriterImpl writer;
```

Comment:
why this change?

Reply:
Not meant to do this, reverted, thanks for reminder.